pkg/trace/obfuscate/obfuscate.go

```diff
@@ -97,6 +98,22 @@ func (o *Obfuscator) Obfuscate(span *pb.Span) {
 	}
 }
 
+// ObfuscateStatsGroup obfuscates the given stats bucket group.
+func (o *Obfuscator) ObfuscateStatsGroup(b *pb.ClientGroupedStats) {
+	switch b.Type {
```

Comment:
Why is obfuscation only implemented for sql and redis?

Reply:
Only those 2 obfuscators address the Resource, the others address specific tags on the Meta map.https://github.com/DataDog/datadog-agent/blob/0a4a94618d169aa09376261ecdba5677f65295a2/pkg/trace/obfuscate/obfuscate.go#L80-L98

Reply:
Got it. Although "cassandra" is treated like "sql" in `Obfuscate` is there a reason to omit it here?

Reply:
I added "cassandra", we can remove it later if it was a mistake.

Reply:
I think you're right. Thanks for adding.